src/qml/QFieldCamera.qml

```diff
@@ -74,62 +132,60 @@ Popup {
       source: camera
 
       autoOrientation: true
+    }
 
-      MouseArea {
-        anchors.fill: parent
+    MouseArea {
+      anchors.fill: parent
 
-        onClicked: {
-          if (camera.lockStatus == Camera.Unlocked)
-            camera.searchAndLock();
-          else
-            camera.unlock();
-        }
+      onClicked: {
+        if (camera.lockStatus == Camera.Unlocked)
+          camera.searchAndLock();
+        else
+          camera.unlock();
       }
+    }
 
-      QfToolButton {
-        id: captureStillImageButton
-        visible: cameraItem.state == "PhotoCapture"
-
-        anchors.right: parent.right
-        anchors.rightMargin: 5
-        anchors.verticalCenter: parent.verticalCenter
+    PinchHandler {
+      enabled: cameraItem.visible && cameraItem.isCapturing
+      target: null
+      acceptedDevices: PointerDevice.TouchScreen | PointerDevice.TouchPad
 
-        round: true
-        roundborder: true
-        bgcolor: "grey"
-        borderColor: Theme.mainColor
+      property real oldScale: 1.0
 
-        onClicked: camera.imageCapture.captureToLocation(qgisProject.homePath+ '/DCIM/')
+      onActiveChanged: {
+        if (active) {
+          oldScale = 1.0
+        } else {
+        }
```

Comment:
Else?

Reply:
@suricactus , else you get nothing :wink: